Source/Transform/KTFractionalFFT.hh

```diff
@@ -0,0 +1,101 @@
+/*
+ * KTFractionalFFT.hh
+ *
+ *  Created on: Mar 8, 2019
+ *      Author: ezayas
+ */
+
+#ifndef KTFRACTIONALFFT_HH_
+#define KTFRACTIONALFFT_HH_
+
+#include "KTProcessor.hh"
+#include "KTData.hh"
+#include "KTSlot.hh"
+#include "KTForwardFFTW.hh"
+#include "KTReverseFFTW.hh"
+
+namespace Katydid
+{
+
+    class KTSliceHeader;
+    class KTProcessedTrackData;
+    class KTTimeSeriesData;
+    class KTFrequencySpectrumDataFFTW;
+
+    /*
+     @class KTFractionalFFT
+     @author E. Zayas
+
+     @brief Performs chirp transform and Fractional FFT via the Garcia algorithm
+
+     @details
+     Separate slots for the chirp transform and Fractional FFT, both of which take a time series data object. The Fractional FFT
+     is performed via chirp -> FFT -> chirp -> Reverse FFT -> chirp, as in Garcia et al. Applied Optics 35, 35. 1996
+     Track slope / rotation angle can be specified directly in the config, or by an incoming track
+     The chirp-only transform is performed in-place. The Fractional FFT is not. The Fractional FFT uses a custom slot function
+     to create a new data pointer, rather than append to the old one. This avoids complications with the ordinary FFT further in the
```

Comment:
Can you explain this to me some more, please?

Reply:
Beefed it up and modified it to reflect changes